pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <jdk.version>17</jdk.version>
+        <jdk.version>11</jdk.version>
```

Comment:
Downgrade of Java version is acceptable in this task but I strongly recommend to uplift Java to version 17 locally. In some of next tasks you will have much more problems during downgrade